spec/features/users/create_users_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+RSpec.feature "Creating users" do
+  scenario "create a valid user" do
+    visit new_user_path
+
+    fill_in "Name", with: "Jane Doe"
+    fill_in "Email", with: "jane@doe.com"
+    click_button "Save"
+
+    expect(current_path).to eq(user_path(User.last))
```

Comment:
I'm assuming `User.last` won't work here. That's fine... for now the test bombs at `click_button "Save"` anyway, when trying to redirect to the show action and get the user by id